### DISCUSSION TITLE
Hyas SEO module help

### DISCUSSION THREAD

**Discussion by pensivedog** [gethyas/hyas#423] [Q&A]
No matter what I try, my pages always have `<meta name=twitter:card content="summary">` instead of `<meta name=twitter:card content="summary_large_image">`.

I have images set in front matter as an array, `images: ["img 1.jpg", img 2.jpg"]`. 

What am I doing wrong?

**Comment by h-enk** [comment]
Look like you found a bug: https://github.com/gethyas/seo/issues/5

**Reply by pensivedog** [reply]
Also, I'm unable to get `twitter:site` or `twitter:creator` to appear in meta data, despite having these set in config.
